starter_templates/.rustc_info.json

```diff
@@ -0,0 +1 @@
+{"rustc_fingerprint":9501504137290250209,"outputs":{"4476964694761187371":["___.exe\nlib___.rlib\n___.dll\n___.dll\n___.lib\n___.dll\nC:\\Users\\syedghufranhassan\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\ndebug_assertions\nproc_macro\ntarget_arch=\"x86_64\"\ntarget_endian=\"little\"\ntarget_env=\"msvc\"\ntarget_family=\"windows\"\ntarget_feature=\"fxsr\"\ntarget_feature=\"sse\"\ntarget_feature=\"sse2\"\ntarget_os=\"windows\"\ntarget_pointer_width=\"64\"\ntarget_vendor=\"pc\"\nwindows\n",""],"1164083562126845933":["rustc 1.36.0 (a53f9df32 2019-07-03)\nbinary: rustc\ncommit-hash: a53f9df32fbb0b5f4382caaad8f1a46f36ea887c\ncommit-date: 2019-07-03\nhost: x86_64-pc-windows-msvc\nrelease: 1.36.0\nLLVM version: 8.0\n",""],"9311443727202421799":["___.exe\nlib___.rlib\n___.dll\n___.dll\n___.lib\n___.dll\nC:\\Users\\syedghufranhassan\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\ndebug_assertions\nproc_macro\ntarget_arch=\"x86_64\"\ntarget_endian=\"little\"\ntarget_env=\"msvc\"\ntarget_family=\"windows\"\ntarget_feature=\"fxsr\"\ntarget_feature=\"sse\"\ntarget_feature=\"sse2\"\ntarget_os=\"windows\"\ntarget_pointer_width=\"64\"\ntarget_vendor=\"pc\"\nwindows\n",""]},"successes":{}}
```

Comment:
I don't think this file needs to be added to the repository, it looks like it has machine-dependent code (I see references to `\Users\syedghufranhassan`). Maybe this should go in the `.gitignore`?

Reply:
yes sir I am shifting it to .gitignore